src/main/java/org/codehaus/mojo/versions/UpdateParentMojo.java

```diff
@@ -95,6 +103,30 @@ public class UpdateParentMojo extends AbstractVersionsUpdaterMojo
                 defaultValue = "false" )
     protected boolean allowDowngrade;
 
+    /**
+     * Whether to allow the major version number to be changed.
+     *
+     * @since 1.2
+     */
+    @Parameter( property = "allowMajorUpdates", defaultValue = "true" )
+    protected boolean allowMajorUpdates = true;
```

Comment:
It is new option for this goal ... so `since` should be `2.13.0`

similar for rest new parameters.